lib/socket.js

```diff
@@ -65,7 +65,7 @@ class Transport extends EventEmitter {
     this._buffer += chunk;
 
     try {
-      this._buffer = jsrs.parseNetworkPackets(this._buffer, packets);
+      this._buffer = json5.parseNetworkPackets(this._buffer, packets);
```

Comment:
I thought of it too, but aim of this PR is simple renaming. I propose to split this into two addons in a follow-up PR.